From d844c643ad45189d14e89cc2002099b4cbf1c80a Mon Sep 17 00:00:00 2001
From: Frank Ramsay <framsay@redhat.com>
Date: Mon, 1 Dec 2014 16:14:50 -0500
Subject: [redhat] RHMAINTAINERS: Add SGI Partner Engineers as maintainers of SGI specific modules

Message-id: <1417450490-15478-1-git-send-email-framsay@redhat.com>
Patchwork-id: 101021
O-Subject: [RHEL 7.1 PATCH] Adding SGI Partner Engineers as maintainers of SGI specific modules.
Bugzilla:
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

Bugzilla: Interal Patch
Upstream Status: Local Only
Build Info: 8313942
Tested: non-code change

  Adding SGI Partner Engineers as maintainers of code specific to SGI hardware.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/RHMAINTAINERS b/redhat/RHMAINTAINERS
index 3d11e24..8b59292 100644
--- a/redhat/RHMAINTAINERS
+++ b/redhat/RHMAINTAINERS
@@ -633,6 +633,21 @@ M: Nikolay Aleksandrov <naleksan@redhat.com>
 S: Maintained
 F: drivers/net/sfc/
 
+SGI SPECIFIC
+M: George Beshers <gbeshers@redhat.com>
+M: Frank Ramsay <framsay@redhat.com>
+M:  Russ Anderson <randerso@redhat.com>
+S: Maintained
+F: drivers/char/uv_mmtimer.c
+F: drivers/misc/sgi-gru/
+F: drivers/misc/sgi-xp/
+F: arch/ia64/include/asm/machvec_uv.h
+F: arch/ia64/include/asm/uv/
+F: arch/ia64/uv/
+F: arch/x86/include/asm/uv/
+F: arch/x86/kernel/apic/x2apic_uv_x.c
+F: arch/x86/platform/uv/
+
 SKGE 10/100/1000 GIGABIT ETHERNET DRIVER
 M: Michal Schmidt <mschmidt@redhat.com>
 S: Maintained
-- 
1.7.1